lib/block-supports/behaviors.php

```diff
@@ -44,6 +44,17 @@ function gutenberg_register_behaviors_support( $block_type ) {
  * @return string                Filtered block content.
  */
 function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
+
+	// We've deprecated the lightbox implementation via behaviors.
+	// While we may continue to explore behaviors in the future, the lightbox
+	// logic seems very specific to the image and will likely never be a part
+	// of behaviors, even in the future. With that in mind, we've rewritten the lightbox
+	// to be a feature of the image block and will also soon remove the block_supports.
+	// *Note: This logic for generating the lightbox markup has been duplicated and moved
+	// to the image block's index.php.*
+	// See https://github.com/WordPress/gutenberg/issues/53403.
+	_deprecated_function( 'gutenberg_render_behaviors_support_lightbox', '17', '' );
```

Comment:
Minor thing. The second argument is documented as the WordPress version, not Gutenberg version. I see we have a couple places where we say something like `'Gutenberg 17.0.0'` instead to avoid confusion.

Reply:
ok, I've just fixed it in 0aba3c4ce53ae2dc59cbf4a6f99028675c697349. 

Thanks for noticing Alex! :)